tests/crypto/test_event_signing.py

```diff
@@ -35,7 +36,9 @@
 
 class EventSigningTestCase(unittest.TestCase):
     def setUp(self):
-        self.signing_key = nacl.signing.SigningKey(SIGNING_KEY_SEED)
+        self.signing_key: signedjson.types.SigningKey = nacl.signing.SigningKey(
```

Comment:
Maybe add a comment above this saying what's happening.

Reply:
Good idea, thanks. What do you make of https://github.com/matrix-org/synapse/pull/11714/commits/683cccbb61c41816d144928ab03d4b1b5bfe83a7 ?

Reply:
Seems reasonable!

Reply:
Cheers. Let's get this merged to make everyone's CI happy.